src/mozanalysis/experiment.py

```diff
@@ -464,8 +458,17 @@ def _build_enrollments_query(self, time_limits, enrollments_query_type):
         """Return SQL to query a list of enrollments and their branches"""
         if enrollments_query_type == "normandy":
             return self._build_enrollments_query_normandy(time_limits)
-        elif enrollments_query_type == "fenix-fallback":
-            return self._build_enrollments_query_fenix_baseline(time_limits)
+        elif (
+            enrollments_query_type == "glean-event"
+            or enrollments_query_type == "fenix-fallback"
+        ):
+            if not self.app_id:
+                raise ValueError(
+                    "App ID must be defined for building Glean enrollments query"
+                )
+            return self._build_enrollments_query_glean_baseline(
+                time_limits, self.app_id
+            )
```

Comment:
I'm seeing enrollment events for Fenix but not for iOS (yet?)

Reply:
Oh interesting; let's check in with Travis on Monday then.